### DISCUSSION TITLE
How suitable for using with Ionic ?  What's things it gives me out of the box ?

### DISCUSSION THREAD

**Discussion by sefatanam** [ant-design/ant-design-mobile#4841] [General]
_I just want to know your thoughts_.

**Comment by awmleer** [comment]
antd-mobile is a component library, while Ionic is a framework which contains UI components, cli, runtime (Capacitor), native APIs. Of course you can use the most part of Ionic and use antd-mobile for writing your application pages. There should be no problem.

If you encounter any bugs with antd-mobile, just open an issue and we'll try to help you.
